### PR TITLE
DOCS: New Tutorials homepage

### PR DESCRIPTION
**Pull request by sgolebiewski-intel**
Updating tutorials homepage and including notebooks generated on 13.09.2022:

https://repository.toolbox.iotg.sclab.intel.com/projects/ov-notebook/0.1.0-latest/20220913220807/dist/rst_files/

### CODE REVIEW

File: docs/tutorials.md [hunk 1]
```diff
@@ -2,28 +2,460 @@
 
 @sphinxdirective
 
+:target:`doxid-tutorials_1md_openvino_docs_tutorials`
```
Comment (xu-yuan1) [inline docs/tutorials.md:5]: do we need this line for md files?

File: docs/tutorials.md [hunk 2]
```diff
@@ -2,28 +2,460 @@
...
 
-@endsphinxdirective
+This collection of Python tutorials are written for running on Jupyter notebooks. 
+The tutorials provide an introduction to the OpenVINO™ toolkit and explain how to 
+use the Python API and tools for optimized deep learning inference. You can run the 
+code one section at a time to see how to integrate your application with OpenVINO 
+libraries.
 
-This collection of Python tutorials are written for running on [Jupyter*](https://jupyter.org) notebooks. The tutorials provide an introduction to the OpenVINO™ toolkit and explain how to use the Python API and tools for optimized deep learning inference. You can run the code one section at a time to see how to integrate your application with OpenVINO™ libraries.
+Notebooks with a |binder logo| button can be run without installing anything.
```
Comment (xu-yuan1) [inline docs/tutorials.md:24]: we can also add a sentence here to mention when I need install notebooks and what's the benefit, with a link to the installation page.

**Comment by xu-yuan1** [comment]
This is the first time I saw the new tutorials homepage. It looks great in general. I had a rough review and add a few comments. Does it need review from @kblaszczak-intel ?

File: docs/tutorials.md (hunk 1 above)
Comment (sgolebiewski-intel) [reply docs/tutorials.md:5]: Actually not. Yeah, the ID of the file is in line no.1 and is generated as "#doxid-tutorials" in the output html. Good catch, thanks :)
```suggestion
```

File: docs/tutorials.md (hunk 2 above)
Comment (sgolebiewski-intel) [reply docs/tutorials.md:24]: Hmm, there is a note below, stating that, I guess :blush:

**Comment by sgolebiewski-intel** [comment]
> This is the first time I saw the new tutorials homepage. It looks great in general. I had a rough review and add a few comments. Does it need review from @kblaszczak-intel ?

Thank you. I believe that having a review from @kblaszczak-intel would be awesome :)

File: docs/tutorials.md (hunk 2 above)
Comment (xu-yuan1) [reply docs/tutorials.md:24]: Great! Sorry I missed that.

File: docs/tutorials.md [hunk 3]
```diff
@@ -2,28 +2,460 @@
...
+Made with `contributors-img <https://contrib.rocks>`__.
+
+
+.. |br| raw:: html
+
+   <br />
+
+.. |chinese-text| raw:: html
+
+   <span style="font-size:10px">的人不一了是他有为在责新中任自之我们</span>
```
Comment (xu-yuan1) [inline docs/tutorials.md:303]: wth is this???

Comment (sgolebiewski-intel) [reply docs/tutorials.md:303]: It's an alias for a directive that injects raw html code. In this case it is a representation of the output text that has been produced by OCR ran by the notebook and is displayed in here:

image

So we have a representation of an input (text in image) and an output (the text) :relaxed:

Are you worried about the meaning of the text?

Comment (xu-yuan1) [reply docs/tutorials.md:303]: aha that explains.....the text in that image is too small for me to tell the correctness of this result, but regarding the Chinese here, I can only say: it's not a valid phrase or sentence. (I know each word but I don't understand any of them when they are put together in this way)  :)

**Review by kblaszczak-intel — APPROVED** [review]
initially agreed to with Jakub Debski from notebooks

**Pull request merged**